libs/application-generic/src/services/calculate-delay/compute-job-wait-duration.service.ts

```diff
@@ -14,8 +14,10 @@ import { ApiException } from '../../utils/exceptions';
 import { isRegularDigest } from '../../utils/digest';
 import { TimedDigestDelayService } from './timed-digest-delay.service';
 import {
-  IChimeraDelayResponse,
   IChimeraDigestResponse,
```

Comment:
Let's remove the Chimera part in a separate PR as well. Right now, we have Chimera, Echo and it feels messy.

Reply:
I think it was already addressed by Denis in another PR.